phoenix-core/src/main/java/org/apache/phoenix/calcite/rules/PhoenixConverterRules.java

```diff
@@ -899,9 +899,6 @@ public static boolean isConvertible(Aggregate input) {
         if (input.getGroupSets().size() > 1)
             return false;
         
-        if (input.containsDistinctCall())
```

Comment:
@maryannxue added a commit to detect for any distinct calls that aren't of type COUNT. Thanks!